.github/workflows/build.yml

```diff
@@ -52,30 +54,36 @@ jobs:
             container: '{ "image": "rippleci/clio_ci:latest" }'
             code_coverage: false
             static: true
+            integration_tests: true
```

Comment:
Why?
Integration tests were on only in nightly on purpose to make PR checks as fast as possible.

Reply:
Yeah, let's not have integration tests run other than in nightly.

Reply:
Thank you, you're right, it's a bug, fixed it 👍